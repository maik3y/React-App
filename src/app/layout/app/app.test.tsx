import * as React from 'react';
import App from './app';
import { shallow } from 'enzyme';

describe('App', (): void => {
  it('should render', (): void => {
    const wrapper = shallow(<App />);
    expect(wrapper.exists()).toBe(true);
  });
});
