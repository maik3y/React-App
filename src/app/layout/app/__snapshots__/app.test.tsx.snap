// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render app 1`] = `
Array [
  <ul>
    <li>
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        /
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        href="/about"
        onClick={[Function]}
      >
        /about
      </a>
    </li>
  </ul>,
  <div>
    <h1>
      Root
    </h1>
    <div>
      <h2>
        Home
      </h2>
      <hr />
      <h3>
        Item 1
      </h3>
      <div>
        <p>
          ID: 
          1
        </p>
        <p>
          Name: 
          Loading
        </p>
        <p>
          State: 
          2
        </p>
      </div>
      <hr />
      <h3>
        Item 2
      </h3>
      <div>
        <p>
          ID: 
          2
        </p>
        <p>
          Name: 
          Loading
        </p>
        <p>
          State: 
          2
        </p>
      </div>
      <hr />
      <h3>
        Item 3
      </h3>
      <div>
        <p>
          ID: 
          3
        </p>
        <p>
          Name: 
          Loading
        </p>
        <p>
          State: 
          2
        </p>
      </div>
      <hr />
      <h3>
        Item 4
      </h3>
      <div>
        <p>
          ID: 
          4
        </p>
        <p>
          Name: 
          Loading
        </p>
        <p>
          State: 
          2
        </p>
      </div>
      <hr />
      <h3>
        Item 5
      </h3>
      <div>
        <p>
          ID: 
          5
        </p>
        <p>
          Name: 
          Loading
        </p>
        <p>
          State: 
          2
        </p>
      </div>
    </div>
  </div>,
]
`;
